workers.py

```diff
@@ -308,9 +338,49 @@ def worker_topology(topology_info: dict) -> None:
 
         logger.debug('%s: Done, exiting', thread.name)
 
+    def worker_host(info: dict) -> None:
+        """Download and forward the content."""
+        thread = current_thread()
+        logger.debug('%s: Worker started', thread.name)
+
+        identity = json.loads(base64.b64decode(b64_identity))
+        account_id = identity.get('identity').get('account_number')
```

Comment:
In case the `identity` scheme changes, this line can fail. If `identity` has no key `'identity'` it would return `None`. Calling `.get('account_number')` would result into complete task failure. Let's change this to at least: 
```suggestion
        account_id = identity.get('identity', {}).get('account_number')
```

Reply:
My did that previously, but then switched back to allow it to error out.
Yeah, shouldn't assume the scheme.

Reply:
I mean.. we shouldn't care what's inside the `b64_identity`. That's platform thing. It's our own initiate we are decoding it. But our service shouldn't fail if there's a sudden change in the scheme. We're not the ones whose setting the rules, so we wouldn't be even told if there's change happening.... So we should rely on it.

Reply:
> Yeah, shouldn't assume the scheme.

by that (and my latest push) I meant an agreement  😉